torch/_library/utils.py

```diff
@@ -373,6 +373,28 @@ def check_aliasing_constraint(name, prev, result, get_module=lambda: "???"):
         storages.add(key)
 
 
+def _c_check_aliasing_constraint(name, args, kwargs, result, get_module=lambda: "???"):
+    """
+    custom operators' outputs must not have any aliases
```

Comment:
can you document here that we only handle Tensors and List[Tensors] in args, kwargs (and nothing else?)